bcda/models/postgres/repository.go

```diff
@@ -509,6 +509,22 @@ func (r *Repository) GetJobKeys(ctx context.Context, jobID uint) ([]*models.JobK
 	return keys, nil
 }
 
+func (r *Repository) GetJobKey(ctx context.Context, jobID uint, fileName string) (*models.JobKey, error) {
+	sb := sqlFlavor.NewSelectBuilder().Select("id", "file_name", "resource_type").From("job_keys")
+	sb.Where(sb.And(sb.Equal("job_id", jobID), sb.Equal("file_name", fileName)))
+
+	query, args := sb.Build()
+	row := r.QueryRowContext(ctx, query, args...)
```

Comment:
Make sure to `close` the row

Reply:
I'm not super familiar with go, I was looking to try to information on closing a single row.  I found this in the documentation which seems to infer that the wrapped `Rows` object is automatically closed after Scan()?

https://github.com/jackc/pgx/blob/v3.6.2/query.go#L23

Reply:
Great find! I was hunting for this in their library and didn't see it, ty ty!